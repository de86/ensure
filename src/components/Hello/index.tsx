import * as React from "react";

export interface HelloProps {
    compiler: string;
    framework: string;
}

export default class Hello extends React.Component<HelloProps, {}> {
    render() {
        return <h1>Yello from {this.props.compiler} and {this.props.framework}!</h1>;
    }
}
