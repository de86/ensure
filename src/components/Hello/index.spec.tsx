import * as React from 'react';
import * as ReactTestRenderer from 'react-test-renderer';

import Hello from './';


test('It should match the Hello snapshot', () => {
    const helloComponent = ReactTestRenderer.create(<Hello compiler="Test" framework="test" />).toJSON();
    expect(helloComponent).toMatchSnapshot('Hello');
});
