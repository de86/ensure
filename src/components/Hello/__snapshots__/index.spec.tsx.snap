// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the Hello snapshot: Hello 1`] = `
<h1>
  Yello from 
  Test
   and 
  test
  !
</h1>
`;
